src/commands/tickets/viewattach.js

```diff
@@ -24,16 +25,86 @@ export default {
+        if (chunks.length === 0) {
+            return await interaction.editReply(await ticketActionMessageObject("No attachments found for this ticket."));
+        }
+
+        const allEmbeds = chunks.map((chunk, index) => {
+            let description = chunk.map((attachment, i) => emojis.redDot + `[Link #${index * chunk.length + i + 1}](${attachment})`).join("\n");
+            return new EmbedBuilder()
+                .setTitle(`🔗 Ticket Attachments for ticket #${ticket.ticketId} (Page ${index + 1}/${chunks.length}) 🔗`)
+                .setDescription(description)
+                .setColor([255, 255, 255]);
+        });
+
+        // Create Buttons
+        const previous_button = new ButtonBuilder()
+            .setCustomId('previous_button')
+            .setLabel('⬅')
+            .setStyle('Primary');
+        const next_button = new ButtonBuilder()
+            .setCustomId('next_button')
+            .setLabel('➡')
+            .setStyle('Primary');
+        const first_button = new ButtonBuilder()
+            .setCustomId('first_button')
+            .setLabel('⬅⬅⬅')
+            .setStyle('Primary');
+        const last_button = new ButtonBuilder()
+            .setCustomId('last_button')
+            .setLabel('➡➡➡')
+            .setStyle('Primary');
+        // Create Action Row
+        const row = new ActionRowBuilder()
+            .addComponents(first_button, previous_button, next_button, last_button);
+
+        let currentPage = 0;
+        await interaction.editReply({ embeds: [allEmbeds[currentPage]], components: [row] });
+
+        // Collectors for buttons, and timeout
+        const filter = i => i.customId === 'previous_button' || i.customId === 'next_button' || i.customId === 'first_button' || i.customId === 'last_button';
+        const msg = await interaction.fetchReply();
+        const collector = msg.createMessageComponentCollector({ filter/*, time: 600000 */}); // 10 minutes timer
+
+        collector.on('collect', async interaction => {
+            if (interaction.customId === 'previous_button' && currentPage > 0) {
+                currentPage--;
+            } else if (interaction.customId === 'next_button' && currentPage < allEmbeds.length - 1) {
+                currentPage++;
+            } else if (interaction.customId === 'first_button') {
+                currentPage = 0;
+            } else if (interaction.customId === 'last_button') {
+                currentPage = allEmbeds.length - 1;
+            }
+            await interaction.update({ embeds: [allEmbeds[currentPage]], components: [row] });
         });
-        const embed = new EmbedBuilder()
-            .setTitle(`🔗 Ticket Attachments for ticket #${ticket.ticketId} 🔗`)
-            .setDescription(ticketAttachments)
-            .setColor([255, 255, 255]); // White
         
-        await interaction.editReply({ embeds: [embed] });
+        collector.on('end', (collected, reason) => {
```

Comment:
collected and reason are declared but never used

Reply:
Given this structure of embed book and collector to kill the buttons is common through the project, this was simply a copy/paste that didn't get slimmed down. Good catch.